### DISCUSSION TITLE
How can I use lazygit outside if nvim

### DISCUSSION THREAD

**Discussion by Muizzyranking** [folke/snacks.nvim#223] [Q&A] [answered]
I really love how snacks lazygit looks and I am wondering if i can use it without launching neovim separately.

**Comment by folke** [comment, marked as answer]
of course you can. Check the snacks source code for the location of the file to append in LG_CONFIG_FILE

**Reply by Muizzyranking** [reply]
oh, worked like charm. You are the man.
